TR-643: Beneficiary-kentälle roolin alkamis- ja päättymispäivät

MikkoKarhunsuo opened the pull request:
Lisätty fin013 Beneficiary-kentälle roolin alkamis- ja päättymispäivät.

MikkoKarhunsuo committed:
TR-643: Lisätty fin013 Beneficiary-kentälle roolin alkamis- ja päättymispäivät.

ext-alexpeltonen reviewed (changes requested):
päivitätkö vielä Dokumentin versio -kohdan riville 9 ja schemas/fin.013.001.02.xsd uuteen versioon (fin.013.001.03.xsd)

Commit:
TR-643: Updated schema fin.013.001 version from 02 to 03

Commit:
Merge branch 'TR-643' of github.com:FinnishCustoms-SuomenTulli/account-register-information-query into TR-643

MikkoKarhunsuo committed:
Update index.md

Päivitetty dokumentin versionumero ja InformationResponseFIN013:n versio 02 -> 03.

MikkoKarhunsuo commented:
Päivitetty dokumentin versionumero ja InformationResponseFIN013 scheman versio.
